application/src/main/java/org/togetherjava/tjbot/features/moderation/scam/ScamBlocker.java

```diff
@@ -251,7 +251,8 @@ private void dmUser(MessageReceivedEvent event) {
     }
 
     private void dmUser(Guild guild, long userId, JDA jda) {
-        jda.openPrivateChannelById(userId).flatMap(channel -> dmUser(guild, channel)).queue();
+        jda.openPrivateChannelById(userId).flatMap(channel -> dmUser(guild, channel)).queue(any -> {
+        }, failure -> logger.debug("failure in Scam blocker Dm  \nTo user id ->" + userId));
```

Comment:
```suggestion
        }, failure -> logger.debug("Unable to send dm message to user {} in guild {}", userId, guild.getId()));
```

better commit my suggestion this time, or else...

Reply:
did it

Reply:
noooooooooooo